bps/bps_ApTemplate.yaml

```diff
@@ -10,11 +10,25 @@ campaign: DM-xxxxx
 
```

Comment:
Similar comments in this file as above, so not to repeating them all here.

Reply:
Same, again 😄 happy to iterate with you on specifics if it would be helpful!